package.json

```diff
@@ -97,11 +97,11 @@
     "a11y:generate": "./scripts/generateAccessibilitySpecs.js",
     "a11y:validate": "./scripts/validateAccessibilitySpecs.js",
     "svgIcons:validate": "./scripts/validateIconSvgSpecs.js",
-    "build": "cd packages/gestalt && yarn build && cd ../gestalt-datepicker && yarn build && cd ../eslint-plugin-gestalt && yarn build",
+    "build": "cd packages/gestalt && yarn build && cd ../gestalt-datepicker && yarn build && cd ../eslint-plugin-gestalt && yarn build && cd ../gestalt-design-tokens && npm run build",
```

Comment:
@ayeshakmaz could we make sure to also use `yarn` (not `npm`) for the `gestalt-design-tokens` package?